source/basic.tex

```diff
@@ -2240,9 +2240,9 @@
 If the \grammarterm{id-expression} in a class member access is a
 \grammarterm{qualified-id} of the form
 \begin{codeblock}
-class-name-or-namespace-name::...
+@\placeholder{class-name-or-namespace-name}@::...
 \end{codeblock}
-the \grammarterm{class-name-or-namespace-name} following the \tcode{.} or
+the \placeholder{class-name-or-namespace-name} following the \tcode{.} or
```

Comment:
@jensmaurer: Wait, did you mean this placeholder to also be printed in (italic) code font?

Reply:
Fixed.